packages/clarity/contracts/taral/exporter-storage.clar

```diff
@@ -1,18 +1,30 @@
 ;; Storage contract for taral exporter
-
+;; NOTE: To be only called by the respective parent contract
 (define-data-var exporter-id-nonce uint u10001)
 
 (define-map exporter-by-principal principal uint)
 
-(define-map exporter-profile {exporter-id: uint} 
-  {
-    name: (string-utf8 100), 
-    category: (string-utf8 100), ;; Merchant /Manufacturer /Service /Project /Deemed exporter
-    orders-next-avail-id: uint
-  }  
+(define-map exporter-profile 
+    {
+        exporter-id: uint,
+        hash: (buff 256)
```

Comment:
Nah, the hash needs to be added in the value side of the map. Two different hashes of the same exporter just mean that the exporter  has changed, but it is still the same exporter. In your case, the `exporter-profile` map could consider the same exporter that has some fields changed as a new exporter, hence, the hash property does not need to be placed in the `key` part of the exporter profile map